### DISCUSSION TITLE
Add Tag or Label

### DISCUSSION THREAD

**Discussion by carakawedhatama** [outline/outline#5968] [Self Hosting] [answered]
Hi,

Is it possible to add `tags` or `label` ? The goals is to search based on the `tags` or `label` that applied .

Thanks !

**Comment by alkanna** [comment, marked as answer]
This is being worked on here : https://github.com/outline/outline/issues/765

**Reply by carakawedhatama** [reply]
thanks for the info @alkanna !

**Reply by Limezy** [reply]
Well... not worked on, only discussed for the moment !
